drivers/power/reset/Kconfig

```diff
@@ -190,6 +190,12 @@ config POWER_RESET_ST
 	help
 	  Reset support for STMicroelectronics boards.
 
+config POWER_RESET_TS_SUPERVISOR
+	bool "embeddedTS Supervisory reset driver"
+	depends on MFD_TS_SUPERVISOR
+	help
+	  Reset support for embeddedTS boards
```

Comment:
vs. a CPU-generated reset? At one point we were experimenting with having the CPU handle the reset. Is leaving this out all there is to it ?-)

Reply:
The CPU can handle the reset via the cpu wdog interface, but when the microcontroller is told to reset directly it tracks the reboot reason.